src/pages/edge/edge-network/tutorials/datastream-config-override-rules.md

```diff
@@ -18,9 +18,9 @@ By default, all **Edge Experience events** are sent to the datastream specified
 * [Configure datastreams](../../../home/getting-started/configure-datastreams.md)
 * [Define overrides in your datastream configuration](https://experienceleague.adobe.com/docs/experience-platform/datastreams/configure.html?lang=en)
 
-## Steps to add datastream config overrides for Lifecycle Edge events using rules
+## Add datastream configuration overrides for Lifecycle Edge events using rules
 
-Let's add **datastream ID override** or **datastream config overrides** for `Lifecycle Edge` events of type **application.launch** and **application.close** by following the next steps.
+Add a **datastream ID override** or a **datastream config overrides** for `Lifecycle Edge` events of type **application.launch** and **application.close** by following the next steps.
```

Comment:
remove second a, reading: or **datastream config overrides**

Reply:
Oh, I thought these were two different types of things, hence why I did that. I'll remove the "a", then.